hip_data_tools/aws/athena.py

```diff
@@ -110,12 +110,16 @@ def _show_result(self, execution_id, max_result_size=1000):
             [[val['VarCharValue'] for val in row['Data']] for row in results['ResultSet']['Rows']])
 
     def _get_query_result(self, execution_id, max_result_size=1000):
-        athena = self.conn.client(self.boto_type)
+        athena = self._client()
         results = athena.get_query_results(QueryExecutionId=execution_id,
                                            MaxResults=max_result_size)
         # TODO: Add ability to parse pages larger than 1000 rows
         return results
 
+    def _client(self):
+        athena = self.conn.client(self.boto_type)
```

Comment:
do you want to instantiate a client everytime this method is called? Should it not be called _get_client? That would closer to the functionality